apps/site/assets/css/_customer-support.scss

```diff
@@ -256,3 +256,13 @@ body {
 .c-route-selector {
   width: 100%;
 }
+
+.ada-container {
+  display: flex;
+  padding-top: 1rem;
+}
+
+.isa-icon {
```

Comment:
question (non-blocking): What does the `isa-` mean here? Is it needed?

Reply:
ISA is the technical name of this icon ♿  Can change to ada to remain consistent with naming throughout pr

Reply:
TIL, thank you! I have no strong preference for changing it then if it is already a conventional name.